ou_dedetai/config.py

```diff
@@ -1,209 +1,1098 @@
+    
+    @property
+    def wine_appimage_recommended_file_name(self) -> str:
+        """Returns the file name of the recommended appimage with extension"""
+        return os.path.basename(self.wine_appimage_recommended_url)
+
+    @property
+    def wine_appimage_recommended_version(self) -> str:
+        # Getting version and branch rely on the filename having this format:
+        #   wine-[branch]_[version]-[arch]
+        return self.wine_appimage_recommended_file_name.split('-')[1].split('_')[1]
+
+    @property
+    def wine_dll_overrides(self) -> str:
+        """Used to set WINEDLLOVERRIDES"""
+        if self._overrides.wine_dll_overrides is not None:
+            return self._overrides.wine_dll_overrides
+        # Default is no overrides
+        return ''
+
+    @property
+    def wine_debug(self) -> str:
+        """Used to set WINEDEBUG"""
+        if self._overrides.wine_debug is not None:
+            return self._overrides.wine_debug
+        return constants.DEFAULT_WINEDEBUG
+
+    @property
+    def wine_output_encoding(self) -> Optional[str]:
+        """Attempt to guess the encoding of the wine output"""
+        if self._overrides.wine_output_encoding is not None:
+            return self._overrides.wine_output_encoding
+        if self._wine_output_encoding is None:
+            self._wine_output_encoding = wine.get_winecmd_encoding(self.app)
+        return self._wine_output_encoding
+
+    @property
+    def app_wine_log_path(self) -> str:
+        if self._overrides.app_wine_log_path is not None:
+            return self._overrides.app_wine_log_path
+        return constants.DEFAULT_APP_WINE_LOG_PATH
+
+    @property
+    def app_log_path(self) -> str:
+        if self._overrides.app_log_path is not None:
+            return self._overrides.app_log_path
+        return constants.DEFAULT_APP_LOG_PATH
+
+    @property
+    def app_winetricks_unattended(self) -> bool:
+        """If true, pass -q to winetricks"""
+        if self._overrides.app_winetricks_unattended is not None:
+            return self._overrides.app_winetricks_unattended
+        return False
+
+    def toggle_faithlife_product_release_channel(self):
+        if self._raw.faithlife_product_release_channel == "stable":
+            new_channel = "beta"
+        else:
+            new_channel = "stable"
+        self._raw.faithlife_product_release_channel = new_channel
+        self._write()
+    
+    def toggle_installer_release_channel(self):
+        if self._raw.app_release_channel == "stable":
+            new_channel = "dev"
+        else:
+            new_channel = "stable"
+        self._raw.app_release_channel = new_channel
+        self._write()
+    
+    @property
+    def backup_dir(self) -> Path:
+        question = "New or existing folder to store backups in: "
+        options = [PROMPT_OPTION_DIRECTORY]
+        output = Path(self._ask_if_not_found("backup_dir", question, options))
+        output.mkdir(parents=True)
```

Comment:
Need to add `exist_ok=True` to `output.mkdir`, otherwise "restore" fails b/c folder already exists.

Reply:
done in https://github.com/FaithLife-Community/LogosLinuxInstaller/pull/206/commits/4f725b6ad0ac7d2647bd05d17fce75887f8089ad